components/evaluate.yaml

```diff
@@ -39,6 +40,14 @@ implementation:
       EOF
       cd workdir/${MAKEDIR}
 
+      if test "${s3_datadir}" != None ; then
+          mkdir -p ./${experiment}/${eval_set}
```

Comment:
eval_set can be a list of evaluation sets separated by a space. I don't think this code handles that.